scripts/ci/run_tests.sh

```diff
@@ -7,4 +7,4 @@ fi
 
 echo "Using ${HADOOP_DISTRO} distribution of Hadoop from ${HADOOP_HOME}"
 
-python test/runtests.py -v
+python test/runtests.py -v $@
```

Comment:
I think the `$@` part is not necessary. `nose.main()` should be able to get `sys.argv` if you add `{posargs:}` to `tox.ini`.